src/register/constants/pbaNumber.ts

```diff
@@ -19,43 +19,53 @@ export const pbaNumber = {
     ],
     groups: [
       {
-        input: {
-          label: {
-            text: 'PBA number 1 (optional)',
-            classes: 'govuk-label--m',
-          },
-          control: 'PBAnumber1',
-          validators: ['pbaNumberPattern', 'pbaNumberMaxLength', 'pbaNumberMinLength'],
-          validationError: {
-            value: 'Enter a valid PBA number',
-            controlId: 'PBAnumber1',
+        fieldset: [
+          {
+            legend: {
+              text: 'What\'s the Payment by Account (PBA) number for your organisation?',
+              classes: 'govuk-fieldset__legend--xl'
+            }
           },
-          classes: 'govuk-!-width-two-thirds',
-        },
-      },
-      {
-        input: {
-          label: {
-            text: 'PBA number 2 (optional)',
-            classes: 'govuk-label--m',
+          {
+            input: {
+              label: {
+                text: 'PBA number 1 (optional)',
+                classes: 'govuk-label--m',
+              },
+              control: 'PBAnumber1',
```

Comment:
Naming convention is a bit off here. Is that what the API dictates? If not, `'pbaNumber1'` or, at a push, `'PBANumber1'` would be more appropriate.

Reply:
changed to PBANumber1